<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QWidget" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление задачи</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AddDialog, #centralwidget {
background: #c29c6b;
}

#submitButton,#cancelButton{
border-style: outset;
border-width: 2px;
border-color: #4a4538;
background:#d6cfbf;
color: black;
vertical-align: middle;
padding: 3px;
}
#submitButton:hover,#cancelButton:hover{
background: #a8a294;
}
#submitButton:pressed,#cancelButton:pressed{
background:  #4a4538;
color: white;
}
</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Название задачи:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="text">
      <string>Приоритет:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="priorityBox">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="deadlineBox">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="text">
      <string>Установить время выполнения задачи</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateTimeEdit" name="deadlineEdit">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="date">
      <date>
       <year>2019</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="delegeteBox">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="text">
      <string>Возможно ли делегировать эту задачу?</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="text">
      <string>Имя исполнителя:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="performerEdit">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="submitButton">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="text">
      <string>Отменить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
